src/index.ts

```diff
@@ -31,13 +31,14 @@ const run = async (): Promise<void> => {
     console.log(prData.coverage.lines.diff);
     console.log(prData.coverage.functions.diff);
     console.log(prData.coverage.branches.diff);
+    console.log(prData.message);
```

Comment:
why are we also logging the message here? I think this shows up in the action?

Reply:
yeah youre right